### ISSUE TITLE
S-164 manual should use updated configuration worksheet

### ISSUE THREAD

**Issue by DavidGrant-NIWC** [iho-ohi/S-164-Sub-Group#42]
The ENCWG has agreed to update S-64 to use a standard configuration checklist for each test. S-164 should use a similar worksheet.

This was proposed at https://iho.int/en/encwg-vtc-2022:

https://iho.int/uploads/user/Services%20and%20Standards/ENCWG/ENCWGVTC_2022/ENCWGVTC1_2022_14_EN_S-64%20Display%20Test%20Template.pdf

![image](https://github.com/iho-ohi/S-164-Sub-Group/assets/62906211/7185d66c-5426-4c91-94f1-93b83e101030)

**Comment by kusala9** [comment]
I don't see a problem accepting this for S-164 - we can talk about it in the meeting. It's quite a large revision to the TDS manual however, but this could be done progressively as the exchange sets are created and baselined. MY only observations would be

* We use exchange sets, not individual datasets (S-64 is described using individual datasets), so propose replacing Dataset with Exchange Set in top right
* Quite a few test reply on changing the system date, there should be somewhere to set this (or just put it in notes).

**Comment by kusala9** [comment]
anyone got this as a Word doc? Easier than making a new one....

**Comment by kusala9** [comment]
we' d need to change "Paper Chart Symbols" to  (not) "Simplified Symbols"....

**Comment by kusala9** [comment]
and symbolized boundaries are "Plain Boundaries = false" ?

**Comment by kusala9** [comment]
agreed at S-164 meeting that a new form will be developed for the next revision of S-164. It was acknowledged that these things are never perfect, but generally, a better stanrdisation of the settings  would be beneficial and this form is a good start point. Word document version received and will be posted on this repo for editing. 

Suggestions for modifications to the template much appreciated, my input is on this thread.

**Comment by DavidGrant-NIWC** [comment]
* Move all context parameters to new section: `Context Parameters`
  * Plain boundaries, Four shades, Full light lines, Shallow water dangers, Simplified symbols, Ignore scale minimum
* Move `Text Groups` to IMS
* There will be additional changes for selecting alert highlights, but I don't have the details yet.

## Additions
|Section|Name|Options|
|:---|:---|:---|
|Mode|-|Route monitoring / Route planning|
|Context parameters|Radar overlay|On / Off|
|Display|Projection|Mercator / Other|
|Display|Legend|On / Off|
|Display|Chart catalog|On / Off|
|Display|Hover|On / Off|

## Modifications (names should match registry entries)
|Section|Old|New|
|:---|:---|:---|
|Context parameters|Show full light lines|Full light lines|
|Context parameters|Paper Chart Symbols|Simplified symbols
|Context parameters|Symbolized Boundaries|Plain boundaries
|Context parameters|SCAMIN Off|Ignore scale minimum
|Context parameters|Unknown objects|Unknown
|IMS|All other text|All other chart text
|Standard Display|Drying lines|Drying line|

**Comment by kusala9** [comment]
look good to me. I will mock one up with the Word doc I have.

**Comment by kusala9** [comment]
updated version shown at last S-164 meeting. More is needed for the Viewing Group Layers I think. Perhaps a drop down of which ones are on/off and the form allows the user to say "All on except ...[named layers]" or "All off except ... [named layers]" which would keep it concise. I think it was pointed out in the meeting that "Other" doesn't have a default meaning (All on or All off) so we need to accomodate both and be clear which applies...

Other things I've missed?

![image](https://github.com/iho-ohi/S-164-Sub-Group/assets/3368156/7f391683-8817-4592-b214-e1502cdb0674)

**Comment by kusala9** [comment]
Highlight Date Dependent is done
Miscellaneous is updated
Radar OVerlay is in context parameters now
Not sure about monitoring/planning, very few tests use it and the instruction can be added to "Setup"
Base/Standard are already in dropdowns for display mode, similarly for Colour Palette
Orientation added under display (not sure how many tests use this either.

![image](https://github.com/iho-ohi/S-164-Sub-Group/assets/3368156/a0cfe212-5c7d-4468-a096-5b5053475dc8)
